experimental/generation/generator/src/library/mergeAssets.ts

```diff
@@ -222,12 +221,10 @@ async function mergeLUFiles(schemaName: string, oldPath: string, newPath: string
         }
     }
 
+    let index = oldText.indexOf(`${os.EOL}[`)
+    let startLines = oldText.substring(0, index)
```

Comment:
This is not really solving the problem, right?  This is just including everything in the file until a [ and this is merging LU files--in theory people can put comments everywhere.

Reply:
Fix the problem.